packages/insured-bridge-relayer/src/MulticallBundler.ts

```diff
@@ -0,0 +1,179 @@
+import winston from "winston";
+import Web3 from "web3";
+
+import { runTransaction, createEtherscanLinkMarkdown } from "@uma/common";
+import { MultiCallerWeb3 } from "@uma/contracts-node";
+import { GasEstimator } from "@uma/financial-templates-lib";
+
+import type { TransactionType, ExecutedTransaction } from "@uma/common";
+import type { Contract } from "web3-eth-contract";
+import type { TransactionReceipt } from "web3-core";
+
+import lodash from "lodash";
+
+interface BaseCall<T extends TransactionType> {
+  transaction: T;
+  message?: string;
+  mrkdwn?: string;
+  level?: string;
+}
+
+type Call = BaseCall<TransactionType>;
+type ExtendedTransasction = TransactionType & { _parent: Contract };
+type ExtendedCall = BaseCall<ExtendedTransasction>;
+
+function isErrorSettlement<T>(input: PromiseSettledResult<T>): input is PromiseRejectedResult {
```

Comment:
This function is also used in `index.ts`, maybe export it there and import here?

Reply:
I can make a helper file and import it.

Reply:
Done.